### ISSUE TITLE
Import error in py36 

### ISSUE THREAD

**Issue by blaggacao** [nubark/facturark#9]
```
(py36) ~/xoe/odoo/facturark (docs-and-tests)  $ facturark --help
Traceback (most recent call last):
  File "/home/blaggacao/xoe/odoo/facturark/.tox/py36/bin/facturark", line 11, in <module>
    load_entry_point('facturark', 'console_scripts', 'facturark')()
  File "/home/blaggacao/xoe/odoo/facturark/.tox/py36/lib/python3.6/site-packages/pkg_resources/__init__.py", line 487, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/blaggacao/xoe/odoo/facturark/.tox/py36/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2728, in load_entry_point
    return ep.load()
  File "/home/blaggacao/xoe/odoo/facturark/.tox/py36/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2346, in load
    return self.resolve()
  File "/home/blaggacao/xoe/odoo/facturark/.tox/py36/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2352, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/blaggacao/xoe/odoo/facturark/.tox/py36/lib/python3.6/site-packages/facturark/__init__.py", line 5, in <module>
    from .api import (build_document, send_document, verify_document,
  File "/home/blaggacao/xoe/odoo/facturark/.tox/py36/lib/python3.6/site-packages/facturark/api.py", line 1, in <module>
    from .analyzer import Analyzer
ModuleNotFoundError: No module named 'facturark.analyzer'
```

This is replicable both on branch `master` and my modified `docks-and-test`.

**Comment by tebanep** [comment]
All the imports in facturark are relative. Are you still experiencing the same issue? You should run the program as a module: python -m facturark

**Comment by tebanep** [comment]
The usage from your Python application, if you are importing the library directly, should be:

facturark.build_document(), facturark.send_document(), etc.

**Comment by blaggacao** [comment]
Ah ok, I see, CLI is still in the works. if you want a refactoring towards Click, maybe I can take that (as I see Click is listed as a dependency)...

**Comment by blaggacao** [comment]
Fixed by #20
